tests/fingerprints_tests.py

```diff
@@ -18,21 +56,165 @@ def test_morgan_fingerprint():
         f"./test_datasets/{'_'.join(dataset_name.split('-'))}/mapping/mol.csv.gz"
     )
     X = dataset["smiles"]
+    X = X[:100]
 
     # Copy X for second sequential calculation
     X_2 = X.copy()
 
     # Concurrent
-    morgan = MorganFingerprintAsBitVect(radius=2, n_bits=2048, n_jobs=-1)
-    X_train = morgan.transform(X)
+    morgan = MorganFingerprint(result_type="default", n_jobs=-1)
+    morgan_hashed = MorganFingerprint(result_type="hashed", n_jobs=-1)
+    morgan_as_bit_vect = MorganFingerprint(
+        result_type="as_bit_vect", n_jobs=-1
+    )
+    X = np.array([Chem.MolFromSmiles(x) for x in X])
+    X_morgan = morgan.transform(X.copy())
+    X_morgan_hashed = morgan_hashed.transform(X.copy())
+    X_morgan_as_bit_vect = morgan_as_bit_vect.transform(X.copy())
+
+    # Sequential
+    X_2 = [Chem.MolFromSmiles(x) for x in X_2]
+    X_seq = np.array([GetMorganFingerprint(x, 2) for x in X_2])
+    X_seq_hashed = np.array([GetHashedMorganFingerprint(x, 2) for x in X_2])
+    X_seq_as_bit_vect = np.array(
+        [GetMorganFingerprintAsBitVect(x, 2) for x in X_2]
+    )
+
+    assert dice_similarity_matrix(X_morgan, X_seq) == 1.0
+    assert dice_similarity_matrix(X_morgan_hashed, X_seq_hashed) == 1.0
+    assert absolute_error(X_morgan_as_bit_vect, X_seq_as_bit_vect) == 0
```

Comment:
Jak ograniczysz się do kilku SMILESów, to możesz explicite sprawdzać jeden po drugim, czy są identyczne. Będzie też szybciej, bo nie będziesz musiał liczyć tych metryk